app-tasks/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
```

Comment:
I couldn't go with slim because I needed `git` so we could track the current commit on master for `botoflow` with this PR.